packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -1043,13 +1092,13 @@ class TextureAndroidViewController extends AndroidViewController {
 
   @override
   Future<void> _sendCreateMessage() async {
-    assert(!_size.isEmpty, 'trying to create $TextureAndroidViewController without setting a valid size.');
+    assert(!_initialSize.isEmpty, 'trying to create $TextureAndroidViewController without setting a valid size.');
```

Comment:
Is `_initialSize` used after creation? Is there a way to refactor the code (without breaking change) so that we don't need this ivar?